app/src/main/java/com/example/recipeopedia/RecipeAdapter.java

```diff
@@ -20,14 +18,13 @@
 
 import org.parceler.Parcels;
 
-import java.sql.Array;
 import java.util.ArrayList;
 import java.util.List;
 
-public class RecipeAdapter extends RecyclerView.Adapter<RecipeAdapter.ViewHolder> implements Filterable {
-    public static final String TAG = "RecipeListActivity";
+public class RecipeAdapter extends RecyclerView.Adapter<RecipeAdapter.ViewHolder> {
+    public static final String TAG = "RecipeAdapter";
     private List<Recipe> mRecipes;
-    Context context;
+    private Context context;
```

Comment:
delete the context reference before landing - you can pass the context into the constructor, just don't save it to a field